cephci/utils/configure.py

```diff
@@ -76,3 +80,36 @@ def setup_ssh_keys(installer, nodes):
                 SSH_COPYID.format(SSH_ID_RSA_PUB, "root", node.hostname),
             ),
         )
+
+
+def accept_eula(node):
+    cmd = "cat" if accept_eula else "touch"
+    cmd += " /usr/share/ibm-storage-ceph-license/accept"
+
+    Package(node).install("ibm-storage-ceph-license", env_vars=accept_eula)
+    return node.exec_command(sudo=True, cmd=cmd, check_ec=True)
+
+
+def exec_cephadm_preflight(installer, build_type, ceph_repo):
+    """Executes cephadm preflight playbook"""
+    try:
+        if not Rpm(installer).query("cephadm-ansible"):
```

Comment:
Change this behavior. Assume cephadm-ansible is installed and have separate method for installing package

Reply:
Suggestion : Add an option cephadm_ansible (bool) in prereq and do the install/check there

Reply:
The changes suggested are made. Now the package check is moved to pre-req and this method now proceeds without checking for packaghe is installed or not